source/codegen/common_helpers.py

```diff
@@ -893,22 +899,22 @@ def get_enum_value_cpp_type(enum: dict) -> str:
 
 
 def is_regular_string_arg(parameter: dict) -> bool:
-    """Excludes: byte arrays"""
+    """Exclude byte arrays."""
     return is_string_arg(parameter) and not parameter["grpc_type"] == "bytes"
 
 
 def is_regular_byte_array_arg(parameter: dict) -> bool:
-    """Excludes: strings and byte arrays that are mapped to enums"""
+    """Exclude strings and byte arrays that are mapped to enums."""
```

Comment:
Same here, I think it should be something like
I think this should be something like
```
"""Return whether the parameter is a "regular" byte array.

This excludes byte arrays that are mapped to enums."""

This excludes byte arrays.
"""
```